assets/js/googlesitekit/api/cache.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * External dependencies
+ */
+import { isEqual } from 'lodash';
+
+let storageBackend;
+/**
+ * Override the storage backend.
+ *
+ * Largely used for tests. Should not be used directly.
+ *
+ * @param {*} backend Backend to set for the cache.
+ */
+export const _setSelectedStorageBackend = ( backend ) => {
+	storageBackend = backend;
+};
+
+let storageKeyPrefix = 'googlesitekit_';
+/**
+ * Override the key prefix used in storage.
+ *
+ * Largely used for tests.
+ *
+ * @param {string} keyPrefix String to prefix storage keys with.
+ */
+export const _setStorageKeyPrefix = ( keyPrefix ) => {
```

Comment:
This is only used for testing and not really necessary. We can just as well test with keys that have the above prefix, especially since that's actually closer to how they are named typically.

Let's remove this function.